Themes/default/languages/EmailTemplates.english.php

```diff
@@ -200,7 +200,7 @@
 $txt['new_announcement_subject'] = 'New announcement: {TOPICSUBJECT}';
 $txt['new_announcement_body'] = '{MESSAGE}
 
-To unsubscribe from these announcements, login to the forum and uncheck "Receive forum announcements and important notifications by email." in your profile.
+To unsubscribe from these announcements, login to the forum and uncheck "Receive forum announcements and important notifications by email." in your profile <em>(Your username > Notificacions > General Settings)</em>.
```

Comment:
```suggestion
To unsubscribe from these announcements, login to the forum and uncheck "Receive forum announcements and important notifications by email." in your profile <em>(Your username > Notifications > General Settings)</em>.
```

Reply:
Addressed in https://github.com/SimpleMachines/SMF2.1/pull/5778/commits/b68a6dcdc21ea1244e9011f7d62c7b20b3b54c50